src/components/manage/Widgets/AlignWidget.jsx

```diff
@@ -1,31 +1,94 @@
+import React from 'react';
+import { defineMessages, useIntl } from 'react-intl';
 import { FormFieldWrapper } from '@plone/volto/components';
-import AlignBlock from '@plone/volto/components/manage/Sidebar/AlignBlock';
-
-/**
- * AlignWidget component.
- * To benefit from styling integration, use with a field named 'align'
- *
- * Example how this field would look in schema:
- *
- * ```jsx
- * {
- *  title: 'Align',
- *  description: 'Layout align',
- *  widget: 'align',
- * }
- * ```
- */
+import { Icon } from '@plone/volto/components';
+import { Button } from 'semantic-ui-react';
+import imageLeftSVG from '@plone/volto/icons/image-left.svg';
+import imageRightSVG from '@plone/volto/icons/image-right.svg';
+import imageFitSVG from '@plone/volto/icons/image-fit.svg';
+import imageNarrowSVG from '@plone/volto/icons/image-narrow.svg';
+import imageWideSVG from '@plone/volto/icons/image-wide.svg';
+import imageFullSVG from '@plone/volto/icons/image-full.svg';
+
+const messages = defineMessages({
+  left: {
+    id: 'Left',
+    defaultMessage: 'Left',
+  },
+  right: {
+    id: 'Right',
+    defaultMessage: 'Right',
+  },
+  center: {
+    id: 'Center',
+    defaultMessage: 'Center',
+  },
+  narrow: {
+    id: 'Narrow',
+    defaultMessage: 'Narrow',
+  },
+  wide: {
+    id: 'Wide',
+    defaultMessage: 'Wide',
+  },
+  full: {
+    id: 'Full',
+    defaultMessage: 'Full',
+  },
+});
+
+export const default_actions_info = ({ intl }) => ({
+  left: [imageLeftSVG, intl.formatMessage(messages.left)],
+  right: [imageRightSVG, intl.formatMessage(messages.right)],
+  center: [imageFitSVG, intl.formatMessage(messages.center)],
+  narrow: [imageNarrowSVG, intl.formatMessage(messages.narrow)],
+  wide: [imageWideSVG, intl.formatMessage(messages.wide)],
+  full: [imageFullSVG, intl.formatMessage(messages.full)],
+});
+
 const AlignWidget = (props) => {
-  const { id, onChange, value, actions } = props;
+  const intl = useIntl();
+
+  const {
+    id,
+    onChange,
+    actions = ['left', 'right', 'center', 'full'],
+    actions_info_map = {},
+    value,
+  } = props;
+
+  React.useEffect(() => {
+    if (!props.value && props.default) {
+      props.onChange(props.id, props.default);
+    }
+  });
+
+  const actions_info = {
+    ...default_actions_info({ intl }),
+    ...actions_info_map,
```

Comment:
@sneridagh Shouldn't these be camel case?

Reply:
Fixed!